packages/devtools_app/test/instance_viewer/instance_viewer_test.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: avoid_redundant_argument_values, false positive on required nullable parameters, import_of_legacy_library_into_null_safe
+// ignore_for_file: false positive on required nullable parameters
```

Comment:
can we remove?

Reply:
done in https://github.com/flutter/devtools/pull/4011